jsk_perception/src/jsk_perception/vil_inference_client.py

```diff
@@ -155,9 +155,9 @@ def topic_cb(self, data):
         vis_msg = ""
         for i, label in enumerate(msg.label_names):
             vis_msg += "{}: {:.2f}% ".format(label, msg.probabilities[i]*100)
-        vis_msg += "\n"
+        vis_msg += "\n\nCosine Similarity\n"
         for i, label in enumerate(msg.label_names):
-            vis_msg += "{}: {:.2f}% ".format(label, msg.label_proba[i]*100)
+            vis_msg += "{}: {:.4f} ".format(label, msg.label_proba[i])
```

Comment:
@tkmtnt7000 
Thank you! Nice comment!
You are absolutely right, the cosine similarity should go from -1 to 1, so there was no need to multiply by 100 for visualization. My porting error. The change to 4 decimal places was to eliminate the 100 multiplier and keep the significant digits.
With the commit I just made, it should be ideal.

Reply:
Thank you! Looks good to me.

Reply:
@Kanazawanaoaki
Shouldn't it be displayed to 6 decimal places?
Originally you were short by 2, so you set it to 4, but since you multiplied by 0.01, so maybe you're back to where you started?

Reply:
@k-okada 
Sorry for the confusion. I was working on another branch, and in that branch I had multiplied by 0.01 and needed 4 digits, but in the PR branch I first forgot to multiply by 0.01 and just used 4 digits.  
The current state is the same as the original working branch.

Reply:
ok, please update PR title and description

Reply:
I have updated the title and description of this PR.

Reply:
ok, please add  link to your ' another branch', that's why people invent WorldWideWeb